/**
 */
package qasvariabilitymodel.tests;

import junit.framework.TestCase;

import junit.textui.TestRunner;

import qasvariabilitymodel.ContainableByQA;
import qasvariabilitymodel.QasvariabilitymodelFactory;

/**
 * <!-- begin-user-doc -->
 * A test case for the model object '<em><b>Containable By QA</b></em>'.
 * <!-- end-user-doc -->
 * @generated
 */
public class ContainableByQATest extends TestCase {

	/**
	 * The fixture for this Containable By QA test case.
	 * <!-- begin-user-doc -->
	 * <!-- end-user-doc -->
	 * @generated
	 */
	protected ContainableByQA fixture = null;

	/**
	 * <!-- begin-user-doc -->
	 * <!-- end-user-doc -->
	 * @generated
	 */
	public static void main(String[] args) {
		TestRunner.run(ContainableByQATest.class);
	}

	/**
	 * Constructs a new Containable By QA test case with the given name.
	 * <!-- begin-user-doc -->
	 * <!-- end-user-doc -->
	 * @generated
	 */
	public ContainableByQATest(String name) {
		super(name);
	}

	/**
	 * Sets the fixture for this Containable By QA test case.
	 * <!-- begin-user-doc -->
	 * <!-- end-user-doc -->
	 * @generated
	 */
	protected void setFixture(ContainableByQA fixture) {
		this.fixture = fixture;
	}

	/**
	 * Returns the fixture for this Containable By QA test case.
	 * <!-- begin-user-doc -->
	 * <!-- end-user-doc -->
	 * @generated
	 */
	protected ContainableByQA getFixture() {
		return fixture;
	}

	/**
	 * <!-- begin-user-doc -->
	 * <!-- end-user-doc -->
	 * @see junit.framework.TestCase#setUp()
	 * @generated
	 */
	@Override
	protected void setUp() throws Exception {
		setFixture(QasvariabilitymodelFactory.eINSTANCE.createContainableByQA());
	}

	/**
	 * <!-- begin-user-doc -->
	 * <!-- end-user-doc -->
	 * @see junit.framework.TestCase#tearDown()
	 * @generated
	 */
	@Override
	protected void tearDown() throws Exception {
		setFixture(null);
	}

} //ContainableByQATest
